apps/blog/src/posts/web-windows-ai/web-windows-ai.md

```diff
@@ -0,0 +1,100 @@
+---
+layout: post
+title: Let's build AI enhanced Progressive Web Apps on Windows
+excerpt: Building AI enhanced PWAs!
+description: This post covers the latest around on-device AI on the Web and how to build AI enhanced PWAs on Windows.
+date: 2024-07-08
+updatedDate: 2024-07-08
+trending: true
+featured: true
+image: posts/whats-new-in-pwa/whats-new-header.png
+isPost: true
+backUrl: '/'
+author:
+  name: Justin Willis
+  twitter: Justinwillis96
+  title: Senior Software Engineer
+  tagline: Seattle based Software Engineer with a passion for the Web
+  image: /author_images/justin_image.jpg
+tags:
+  - post
+  - AI
+  - PWAs
+---
+
+# Build PWAs on and for Windows!
+
+Are you a Web Developer? Do you use Windows or are you interested in switching to Windows? Let's dive into how you can use Windows to have the **BEST Web Development experience**. We will set up a dev environment with **WSL (Windows Subsystem for Linux)** and discuss how it brings a familiar dev environment to your PC. Then, we'll explore the details of how **PWAs (Progressive Web Applications)** work on Windows. Finally, if you make it to the end, we'll quickly get started building a new PWA on Windows!
+
+## Setting up your Dev Environment
+
+### Installing Windows Subsystem for Linux (WSL)
+
+WSL allows developers to install a Linux distribution (such as Ubuntu, Debian, Arch, etc.) and use Linux applications, utilities, and command-line tools directly on Windows, with low overhead. To install WSL:
+1. Open PowerShell in administrator mode.
+2. Enter the following command: `wsl --install`
+3. Restart your machine.
+4. This command will enable the necessary features to run WSL and install the Ubuntu distribution of Linux.
+5. The first time you launch a newly installed Linux distribution, a console window will open, and you'll see a message asking you to wait for the distro to finish installing.
+
+## Why WSL?
+
+WSL enables several key experiences for web developers:
+- Node.js and other Node-based tools are sometimes built with the assumption that there will be a Unix-based environment, which means they may not work well on Windows. WSL fixes this.
+- Node.js can be slow on Windows, especially around NPM installs. WSL provides a solution to this.
+- WSL allows you to test your PWA in browsers on Linux, right from your PC. Learn more
+
+## Progressive Web Applications on Windows
+
+Progressive Web Applications are first-class on Windows. PWAs
+- Are deeply integrated, with PWAs in the Microsoft Store, showing up in normal app settings, synced across your Windows devices and more
```

Comment:
Reading this first item on the list was weird to me
PWAs
- Are deeply integrated, with PWAs in the Microsoft Store, showing up in normal app settings, synced across your Windows devices and more